src/configure/resources/tracePoints.ts

```diff
@@ -0,0 +1,34 @@
+export class TracePoints {
+    // Combined telemetry trace points
+    public static telemetryKey: string = '';
+    public static CurrentFunction: string = 'currentFunction';
+    public static RepoProvider: string = 'repoProvider';
+    public static AzureLoginRequired: string = 'azureLoginRequired';
+    public static Command: string = 'command';
+    public static JourneyId: string = 'journeyId';
+    public static EntryPoint: string = 'entryPoint';
+    public static SourceRepoLocation: string = 'sourceRepoLocation';
+    public static NewOrganization: string = 'newOrganization';
+    public static ChosenTemplate: string = 'chosenTemplate';
+    public static PipelineDiscarded: string = 'pipelineDiscarded';
+    public static ViewPipelineClicked: string = 'viewPipelineClicked';
+    public static GitHubPatStartTime = 'gitHubPatStartTime';
+    public static GitHubPatEndTime = 'gitHubPatEndTime';
+    public static CommandStartTime = 'commandStartTime';
+    public static CommandEndTime = 'commandEndTime';
+    public static OrganizationListDuration = 'organizationListDuration';
```

Comment:
is this for measuring performance?

Reply:
Yes. As mentioned in the above comment :https://github.com/hiyadav/azure-pipelines-vscode/pull/6/files#r315582413 
We can discuss a general solution.